internal/sirius/get_task_list.go

```diff
@@ -165,146 +164,92 @@ func CreateFilter(taskTypeSelected []string, selectedAssignees []string, dueDate
 	return strings.TrimRight(filter, ",")
 }
 
-func SetTaskTypeName(v []ApiTask, loadTasks []ApiTaskTypes) []ApiTask {
-	var list []ApiTask
-	for _, s := range v {
-		task := ApiTask{
-			ApiTaskAssignee: CaseManagement{
-				CaseManagerName: GetAssigneeDisplayName(s),
-				Id:              GetAssigneeId(s),
-				Team:            GetAssigneeTeams(s),
-			},
-			ApiTaskDueDate:          s.ApiTaskDueDate,
-			ApiTaskId:               s.ApiTaskId,
-			ApiTaskHandle:           s.ApiTaskHandle,
-			ApiTaskType:             s.ApiTaskType,
-			TaskTypeName:            GetTaskName(s, loadTasks),
-			ClientInformation:       GetClientInformation(s),
-			ApiPriorityTask:         s.ApiPriorityTask,
-			CalculatedDueDateColour: GetCalculatedDueDateStatus(s.ApiTaskDueDate, time.Now),
-		}
-		list = append(list, task)
+func (t Task) GetDueDateStatus(now ...time.Time) DueDateStatus {
+	removeTime := func(t time.Time) time.Time {
+		y, m, d := t.Date()
+		return time.Date(y, m, d, 0, 0, 0, 0, time.Local)
```

Comment:
Is there a reason you chose time.Local over time.Now. I've used it before for altering the time for daylight saving in the deputy-hub but I don't _think_ it would do that here as we've not set a timezone in the dockerfile.

Reply:
Hmm the two aren't interchangeable though... time.Now() is a method that creates a Date, whereas time.Local is a Location struct.

I've used `time.Now()` to get the current date and time, and then passing that to `removeTime()`, which in turn will create a new date with the hours/minutes/seconds set to 0. Are you suggesting I should use the Location from the original Date? i.e. replacing `time.Local` with `t.Location()`?

Reply:
Ah maybe I misunderstood, if you're always removing all of the time I don't think it's an issue.